components/CommonComponents/Banner/index.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { useMemo } from 'react';
+import { sanitize } from 'isomorphic-dompurify';
+import { dateIsBetween } from '../../../util/dateIsBetween';
+import { isAbsoluteUrl } from '../../../util/isAbsoluteUrl';
+import styles from './index.module.scss';
+import { WebsiteBanner } from '../../../types';
+
+export interface BannerProps {
+  bannersIndex: WebsiteBanner;
+}
+
+const useTextContent = ({ text, link, bannerBtnText }: WebsiteBanner) =>
+  useMemo(() => {
+    if (text) {
+      return (
+        <p>
+          <a href={link} target="_blank" rel="noopener noreferrer">
+            <button type="button">{bannerBtnText || 'Read More'}</button>
```

Comment:
This is invalid HTML as `<button>` can't be nested in a `<a>`

Reply:
This is fixed